Assets/Scripts/Audio.gd

```diff
@@ -0,0 +1,29 @@
+extends Node
+
+var player
+var popup
+var Settings_Sound = load("res://Assets/Scripts/Settings_Sound.gd").new()
```

Comment:
Użuwaj autoload'a w projektcie, kiedy trzeba sherować resource'y w całym projektcie.

Reply:
To jest do wywalenia